src/styles/Login.css.js

```diff
@@ -0,0 +1,49 @@
+import { css } from 'react-emotion';
+
+export const pageStyles = css`
+  min-width: 320px;
+  height: 100vh;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const formStyles = css`
+  display: grid;

```

Comment:
I need to double check that the tablets our techs have in the field will run a browser that supporters css grid https://caniuse.com/#feat=css-grid. I'm pretty sure its Chrome.